IdentityCore/src/workplacejoin/mac/MSIDWorkPlaceJoinUtil.m

```diff
@@ -158,4 +158,17 @@ + (SecIdentityRef)copyWPJIdentity:(__unused id<MSIDRequestContext>)context
     return identityRef; //Caller must call CFRelease
 }
 
++ (nullable NSString*)getWPJStringData:(id<MSIDRequestContext>)context
+                          identifier:(nonnull NSString*)identifier
+                          error:(NSError*__nullable*__nullable)error
+{
+    // Building dictionary to retrieve given identifier from the keychain
+    NSMutableDictionary *query = [[NSMutableDictionary alloc] init];
```

Comment:
Looks like query between iOS and macOS is the same except access group. Possible to combine?